src/VisionModel/resnet.jl

```diff
@@ -0,0 +1,79 @@
+using Flux
+
+function basicblock(inplanes::Int, planes::Int, stride::Int, dilation::Int)
+  basic =  Chain(
+      Conv((3, 3), inplanes=>planes, stride=stride, pad=(1,1), dilation=dilation),
+      BatchNorm(planes, λ = relu),
+      Conv((3, 3), planes=>planes, stride=stride, pad=(1,1), dilation=dilation),
+      BatchNorm(planes, λ=relu)
+    );
```

Comment:
We'll want to handle the downsampling case as well. For the second `Conv`, `stride == 2` when downsampling and `stride == 1` otherwise. [Refer to this PR](https://github.com/FluxML/Metalhead.jl/blob/069a4ea2575aa30ca063b4156da4864dee6d96bc/src/resnet.jl) for guidance.

Also, we don't want to set `dilation`, because that is a design artifact from PyTorch's implementation.